### ISSUE TITLE
Reading time below 1 min is off by up to 1 min

### ISSUE THREAD

**Issue by technovangelist** [Kirozen/vsce-wordcounter#8]
Great plugin

I have reading time set to 145 (for voiceovers). When i am still approaching 145, reading time should be less than 1 minute. It looks like the seconds are accurate, but 1 min is added to it. So when there are 50 seconds on content, it will read 1 min 50 seconds. Every thing is accurate after it reaches 1 minute

**Comment by Kirozen** [comment]
Fixed in future release
